pkg/kubelet/lifecycle/handlers_test.go

```diff
@@ -257,3 +257,460 @@ func TestRunHandlerHttpFailure(t *testing.T) {
 		t.Errorf("unexpected url: %s", fakeHTTPGetter.url)
 	}
 }
+
+func TestNoNewPrivsRequired(t *testing.T) {
+	admitPrivEsc := false
+	for _, testCase := range []struct {
+		name     string
```

Comment:
this looks unused, so either please use it or drop it

Reply:
fixed